settings.gradle

```diff
@@ -2,7 +2,7 @@ pluginManagement {
 	repositories {
 		maven {
 			name = 'Fabric'
-			url = 'https://maven.fabricmc.net/'
+			url = 'https://maven2.fabricmc.net/'
```

Comment:
this was just because of my, for some reason slow, internet, can probably be changed back? Downloading the Yarn JD jar took 8 minutes :concern:

Reply:
Tried this on my machine and had no troubles with `maven.fabricmc.net` - I think we can change it back :)